docs/ENV.md

```diff
@@ -1,6 +1,6 @@
 ## Environment variables
 
-This guide explains all the available configurations in your `.env` file so that you can configure your environments accordingly.
+This guide explains all the available configurations in your `.env` file so that you can configure your environments accordingly. For more details, you can also check official documentation from Cube.js on [Environment Variables](https://cube.dev/docs/reference/environment-variables).
```

Comment:
the link talks about this env variable:
```
CUBEJS_APP | An application ID used to uniquely identify the Cube.js deployment. Can be different for multitenant setups. Defaults to cubejs
```
Wouldn't we need this for our multitenancy part?

Reply:
I believe this is for multitenant set up of cubejs.